src/main/java/com/hubspot/jinjava/interpret/JinjavaInterpreter.java

```diff
@@ -259,18 +259,46 @@ public String renderFlat(String template) {
    * @return rendered result
    */
   public String render(String template) {
-    return render(parse(template), true);
+    return render(template, config.getMaxOutputSize());
+  }
+
+  public String render(String template, long renderLimit) {
+    return render(parse(template), true, renderLimit);
   }
 
   /**
-   * Render the given root node, processing extend parents. Equivalent to render(root, true)
+   * Render the given root node, processing extend parents. Equivalent to render(root, true, -1)
```

Comment:
Don't need to change this, also it's not the same as that since it passes the max output size from the config